modules/linux

```diff
@@ -31,6 +31,9 @@ linux_hash := a6fbd4ee903c128367892c2393ee0d9657b6ed3ea90016d4dc6f1f6da20b2330
 else ifeq "$(CONFIG_LINUX_VERSION)" "5.10.5"
 linux_version := 5.10.5
 linux_hash := 3991a9e16a187d78d5f414d89236ae5d7f404a69e60c4c42a9d262ee19612ef4
+else ifeq "$(CONFIG_LINUX_VERSION)" "6.1"
```

Comment:
Would be nice if you could reuse version 6.1.2 as of purism otherwise time of builds on circleci will explode for no good reason. Make sure to save your Linux config in oldefconfig format.

https://github.com/osresearch/heads/pull/1419/files#diff-C44C192F63AAB4E550C64696EA703787R34

Reply:
ok, can do, this means we'll wait at least until #1419 is upstreamed, then I'll rebase and adapt on top of that ?

Reply:
or should I just pull in the 6.1.8 kernel changes and make this PR standalone + 6.1.8-based ?

Reply:
@daringer you can adapt your code and test on top of that kernel version is what I meant. The commit related to your kernel version bump for the kernel module, it same then Purism PR, will be easy to drop from a dynamic rebase 

`git rebase -i commit_prior_of_that_one` would do and fasten the process.
#1419 is being tweaked with last bits prior of merge, so if you tested it against your own branch, that will be next version of Heads. Most important steps for Nitrokey here is being aware of what Restricted Boot/Basic Boot is and test if their newer fbwhiptail with AVX works as well for you where no regression were observed on x230. Next step there will be to move away of i915drmfb in place of simplefb so we have an easy portability to other platforms in the future not having to hack our way around and tainting the kernel and being a step ahead in legacy boot deprecation that is in the air which would not affect us.

TLDR: please test #1419 @daringer :) If your board depend on that kernel version and that PR works, that will be merged soon.

Reply:
rebased on master, thus using 6.1.8 now